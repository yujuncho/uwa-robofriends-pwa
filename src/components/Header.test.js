import { shallow } from "enzyme";
import Header from "./Header";

describe("Header", () => {
  it("render", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });

  it("shouldComponentUpdate returns false", () => {
    const wrapper = shallow(<Header />);
    const instance = wrapper.instance();
    expect(instance.shouldComponentUpdate()).toBe(false);
  });
});
