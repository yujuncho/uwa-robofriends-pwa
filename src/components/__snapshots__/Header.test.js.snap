// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render 1`] = `
<div>
  <h1
    className="f1"
  >
    RoboFriends
  </h1>
  <CounterButton />
</div>
`;
